var/spack/repos/builtin/packages/py-neurodamus/package.py

```diff
@@ -13,6 +13,7 @@ class PyNeurodamus(PythonPackage):
     git      = "ssh://bbpcode.epfl.ch/sim/neurodamus-py"
 
     version('develop', branch='master')
+    version('report_flush', branch='sandbox/jblanco/reports_flush', preferred=True)
```

Comment:
@jorblancoa  @ferdonline  : sandbox branch of py-neurodamus in production?

Reply:
my fault, will change it back (it was supposed to be only libsonata change....